crates/postgres-introspection/src/config.rs

```diff
@@ -1,6 +1,6 @@
 use grafbase_database_definition::TableWalker;
 use serde::Deserialize;
-use std::collections::HashMap;
+use std::collections::BTreeMap;
```

Comment:
These are all to fix flaky tests, not related to the relay PR, just annoying and wanted to fix.